sidebars.js

```diff
@@ -634,7 +634,8 @@ module.exports = {
         {
           "Helm/Kubernetes": [
             "self-managed/platform-deployment/helm-kubernetes/overview",
-            "self-managed/platform-deployment/helm-kubernetes/deployment",
+            "self-managed/platform-deployment/helm-kubernetes/deploy",
+            "self-managed/platform-deployment/helm-kubernetes/upgrade",
```

Comment:
Due to the fact that we now have separate versions for Optimize than the rest of the docs, parallel sidebar changes will need to be made at https://github.com/camunda/camunda-platform-docs/blob/main/optimize_sidebars.js#L1153-L1156.

Reply:
I took care of this in the latest commit.